csr.py

```
@@ -241,7 +241,11 @@
         valid = {}
         fields = ('C', 'ST', 'L', 'O', 'OU', 'CN', 'keySize', 'keyType', 'curve', 'subjectAltNames', 'allowPrivateDomains')
         required = ('CN',)
-        
+
+        # Maximum limits for safety
+        MAX_SANS_COUNT = 100
```

Ruff (0.11.9)

Local variable `MAX_SANS_COUNT` is assigned to but never used

Remove assignment to unused variable `MAX_SANS_COUNT`

(F841)